framework/web/UrlManager.php

```diff
@@ -488,4 +488,16 @@ public function setHostInfo($value)
     {
         $this->_hostInfo = rtrim($value, '/');
     }
+
+    /**
+     * Remove numeric indexes from the response
+     * @param $params
+     *
+     * @return mixed|string
+     */
+    protected function custom_http_build_query($params) {
+        $request = http_build_query($params);
+        $request = preg_replace('#%5B[0-9]%5D#', '%5B%5D', $request);
```

Comment:
For example, `http_build_query(['one' => 'two[1]'])` returns _one=two%5B1%5D_ where the value would be incorrectly modified. That's why I opt for a custom implementation of `http_build_query` where this can be easily avoided.